### PR TITLE
Tidy up templates

### PR DESCRIPTION
**Pull request by sdbernard**
Lots of little tweaks in this PR
• Add IDs to lines for Illustrator
• Add labels option for bars/columns
• add circleSize multiplier to allow adjustment of circle size on circle timeline
• move data manipulation out of index.js and into parseData.js on line chart
• add css for non-responsive label
• make bar/column templates more consistent

### CODE REVIEW

File: column-grouped/columnGroupedChart.js [hunk 1]
```diff
@@ -11,6 +11,7 @@ export function draw() {
     let markers = false; // eslint-disable-line
     let includeMarker = undefined; // eslint-disable-line
     let interpolation = d3.curveLinear;
+    let numbers = true;
```
Comment (aendrew) [inline column-grouped/columnGroupedChart.js]: Comments explaining what these do could be useful, `numbers` isn't a very descriptive variable name (could maybe be referred to as `showNumberLabels` or some such, possibly?).

File: column-grouped/columnGroupedChart.js [hunk 2]
```diff
@@ -115,6 +132,10 @@ export function draw() {
         }
         return chart;
     };
+    chart.numbers = (d) => {
```
Comment (aendrew) [inline column-grouped/columnGroupedChart.js]: In general we should generally make setters also getters, it's an extra line and improves interoperability down the line.

File: column-grouped/parseData.js [hunk 3]
```diff
@@ -9,23 +9,42 @@ import * as d3 from 'd3';
...
+                const plotData = data.map(d => ({
+                    name: d.name,
+                    groups: getGroups(seriesNames, d),
+                }));
+
+                if (sort === 'descending') {
+                    plotData.sort((a, b) =>
+                    // console.log("sortON=",sortOn)
+                    // console.log("SortOn",a.groups[sortOn],a.groups[sortOn].value,b.groups[sortOn],b.groups[sortOn].value)
+                        b.groups[sortOn].value - a.groups[sortOn].value);// Sorts biggest rects to the left
```
Comment (aendrew) [inline column-grouped/parseData.js]: Probably worth removing the extraneous commented code above.

Comment (sdbernard) [reply column-grouped/parseData.js]: done

File: column-grouped/columnGroupedChart.js (hunk 2 above)
Comment (sdbernard) [reply column-grouped/columnGroupedChart.js]: You mean

File: column-grouped/columnGroupedChart.js (hunk 1 above)
Comment (sdbernard) [reply column-grouped/columnGroupedChart.js]: Ahh sorry I only put the comment in the index.js file, added

**Pull request merged**